core/sys/types/build.rs

```diff
@@ -24,11 +24,15 @@ fn sgx_library_path() -> String {
 
 fn main() {
     let bindings = Builder::default()
-        .header_contents("core_types.h", "#include <sgx_error.h>")
```

Comment:
What's the value of doing `.header_contents("core_types.h", "#include <sgx_error.h>")` vs doing `.header("sgx_error.h")`? Is bindgen unable to find `sgx_error.h`?

Reply:
I did see that, and I agree with you in that I don't particularly care how we go about this.